scripts/scil_compute_local_tracking.py

```diff
@@ -13,6 +13,27 @@
 Algo 'prob': a direction drawn from the empirical distribution function defined
 from the SF.
 
+The local tracking algorithm can also run on the GPU using the --use_gpu
+option (experimental). Below is a list of known divergences between the
+CPU and GPU implementations:
+    * Backend: The CPU implementation uses DIPY's LocalTracking and the
+        GPU implementation uses an in-house OpenCL implementation.
+    * Algo: For the GPU implementation, the only available algorithm is
+        Algo 'prob'.
+    * GPU options: Nearest neighbour interpolation is implemented for
+        GPU tracking and can be enabled with '--sh_interp nearest'.
+        Backward tracking can be disabled using '--forward_only.
+
+Streamlines are filtered by minimum length, but not by maximum length. This
+means that streamlines are stopped and returned as soon as they reach the
+maximum length instead of being discarded if they go above the maximum allowed
+length. This allows for short-tracks tractography, where streamlines are
+prematurely stopped once they reach some target length.
+
+However, for this reason, there may be streamlines ending in the deep white
+matter. In order to use the resulting tractogram for analysis, it should be
```

Comment:
We should have this as a warning message when you run --gpu

Reply:
The CPU implementation now behaves the same, there is a weird detail in DIPY implementation where `maxlen` is actually the maximum length per pass (forward and backward) and not the total maximum length. If we don't like this behaviour I can change it for both, or add an argument `--cut_to_max_length` to toggle this feature.